test/helpers/kubectl.ts

```diff
@@ -72,7 +72,7 @@ export async function applyK8sYaml(pathToYamlDeployment: string): Promise<void>
 
 export async function createPodFromImage(name: string, image: string, namespace: string) {
   console.log(`Letting Kubernetes decide how to manage image ${image} with name ${name}`);
-  await exec(`./kubectl run ${name} --image=${image} -n ${namespace} -- sleep 999999999`);
+  await exec(`./kubectl run ${name} --generator=run-pod/v1 --image=${image} -n ${namespace} -- sleep 999999999`);
```

Comment:
so this is backwards compatibility for the older version of kubectl the OS3 tests will use, correct? where the default generator was deployment rather than pod?
I wonder if it's worth mentioning more explicitly somehow, but up to you